docs/view_components/welcome.rst

```diff
@@ -44,7 +54,7 @@ Link with other view-components
 
 
 :ref:`menu_bar-label`
-     User can re-enable appearance of the welcome view-component through options within the file menu
+     User can re-enable appearance of the welcome view-component through options within the *Display* or *Preferences* menu.
```

Comment:
repetition with requirement 3, we can put it just here as it is not really a requirement
```suggestion
     The user can later re-enable the appearance of the pop-up window by clicking *Display welcome view* in *Preferences* from the menu.
```

Reply:
I would keep the term view-component though, it is more general as "pop-up" (a pop up is a window which pops over another window, but our view component could as well be a full view or a widget (like a window within the main window, it cannot go out of the main window frame, while a pop-up could go out of the main window frame)

Reply:
Sure thing